nginx/Dockerfile

```
@@ -13,17 +13,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# https://github.com/kubernetes/ingress-nginx
-FROM k8s.gcr.io/ingress-nginx/controller:v1.12.6
+FROM cgr.dev/astronomer.io/ingress-nginx-controller:1.13-dev
```

We should pin this further so we can have an accurate version number for our image.
```suggestion
FROM cgr.dev/astronomer.io/ingress-nginx-controller:1.13.5-dev
```